### PR TITLE
added opta-stats documentation

### PR DESCRIPTION
**Pull request by JuanAlejandro**
Added OptaStats documentation.

### CODE REVIEW

**Review by arturoapplicaster — CHANGES_REQUESTED** [review]
Change images URL's and change the word whole for another term

File: Opta-Stats/OptaStats.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+-- `all_teams_screen`
+-- `team_screen`
+-- `player_screen`
+- `<other_data_key>` **(Required for specific screens)**: To show values from a specific entity:
+-- `match_id=<op_id>&push=<true_or_false>`
+-- `team_id` (`team_screen` and `all_matches_screen`)
+-- `player_id` (`player_screen`)
+
+#### Match details & Push option
+When opening match details screen through URL scheme, there is another key named push where indicates if the URL scheme comes from the push or if it’s being opened through the regular way.
```
Comment (casajavi) [inline Opta-Stats/OptaStats.md]: When opening match details screen through URL scheme, there is another key named `push` ~where~ **which** indicates if the URL scheme comes from the push or if it’s being opened ~through the regular way~ within the app.

File: Opta-Stats/OptaStats.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Opta Stats
+
+*Built by LATAM Team*
+
+**Supports:** *iOS and Android*
+*Current Version: 0.2.22 (Android), 1.0.52 (iOS)*
+
+## About
+The Opta Statistic Plugin is being developed primarily to show the statistics for Copa America 2019 tournament. Since Opta is the main provider for several events in LATAM we are pivoting this plugin to be general.
```
Comment (casajavi) [inline Opta-Stats/OptaStats.md]: ... we are pivoting this plugin to be a generic Opta plugin.

File: Opta-Stats/OptaStats.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+## URL Schemes supported
+We defined a set of pre-configured number of screens which you can open through a URL scheme from anywhere in the application. In order to do so, we need to define a well-known URL schema. These are the proposed schemas for this integration.
```
Comment (casajavi) [inline Opta-Stats/OptaStats.md]: replace "We defined a set of pre-configured number of screens" with "We predefined a set of screens"

File: Opta-Stats/OptaStats.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+-- `player_screen`
+- `<other_data_key>` **(Required for specific screens)**: To show values from a specific entity:
+-- `match_id=<op_id>&push=<true_or_false>`
+-- `team_id` (`team_screen` and `all_matches_screen`)
+-- `player_id` (`player_screen`)
+
+#### Match details & Push option
+When opening match details screen through URL scheme, there is another key named push where indicates if the URL scheme comes from the push or if it’s being opened through the regular way.
+
+When the url scheme is coming from push the `match_id` is different from the one provided by Opta. Let's call this data, `match_id_from_push`. With this id provided by the push is necessary to send to Opta API as `match_id`, `urn:perform:opta:fixture:<match_id_from_push>`. The id coming from the push may have a letter at the beginning. It is necessary to remove this letter to use it as `match_id` in the API call.
```
Comment (casajavi) [inline Opta-Stats/OptaStats.md]: ~With this id provided by the push is necessary to send to Opta API as `match_id`, `urn:perform:opta:fixture:<match_id_from_push>`.~ replace with: "The match id from the push must be sent to the Opta API as `match_id`, `urn:perform:opta:fixture:<match_id_from_push>`"

File: Opta-Stats/OptaStats.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+![Team screen preview](media/team_screen.png)
+
+### <a name=player-screen></a> Player Screen
+You can find the whole design [here](https://app.zeplin.io/project/5c8863d4039b9f39096d453d/screen/5cab7ce9e89d3f338980bcb3).
+
+![Player screen preview](media/player_screen.png)
+
+## Plugin configuration
+In order to make this plugin general and that can be used for other tournaments, we define the following parameters:
```
Comment (casajavi) [inline Opta-Stats/OptaStats.md]: In order to make this plugin ~general~ generic and ~that can~ be used for other tournaments, we define**d** the following parameters:

**Review by casajavi — CHANGES_REQUESTED** [review]

**Review by arturoapplicaster — APPROVED** [review]

**Review by casajavi — APPROVED** [review]

**Pull request merged**